app/fileManager.js

```diff
@@ -86,7 +86,7 @@ export default function(state, emitter) {
     document.addEventListener('blur', () => (updateTitle = true));
     document.addEventListener('focus', () => {
       updateTitle = false;
-      emitter.emit('DOMTitleChange', 'Firefox Send');
+      emitter.emit('DOMTitleChange', state.title);
```

Comment:
This isn't what we need here. The `DOMTitleChange` event sets `state.title` to the passed argument, so this effectively does nothing. A good way to do this is to add a config parameter to the server and pass it through to the frontend.

Reply:
Could you elaborate on that alternative?

Reply:
We'll probably need:
- a new config parameter / variable in `server/config.js` for `PRODUCT_NAME`
- add said variable to `server/routes/jsconfig.js`
- add that variable to `state` in `app/main.js`
- also add it to `server/state.js`
- change the hard coded strings